conf/etc/cron.hourly/ha_logrotate

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Avoid creating multiple custom files.
Use `https://github.com/Seagate/cortx-prvsnr/pull/622` to avoid any files under `/etc/cron.hourly` and just add files under `/etc/logrotate_hourly.d`